src/components/Match/Laning/index.jsx

```diff
@@ -20,17 +20,17 @@ class Laning extends React.Component {
     this.setState({ ...this.state, selectedPlayer: playerSlot });
   }
   render() {
-    const players = [...this.props.match.players];
+    const { match } = this.props;
```

Comment:
are these changes needed?

Reply:
Oh this change is to revert the code back to original state. If you check out all the change, it was only a copy of array xD.